src/input/scroll.h

```diff
@@ -20,12 +20,11 @@ void scroll_restore_margins(void);
 int scroll_get_delta(const mouse *m, pixel_offset *delta, scroll_type type);
 
 void scroll_drag_start(int is_touch);
-void scroll_drag_move(void);
 int scroll_drag_end(void);
 
-void scroll_arrow_left(int is_down);
-void scroll_arrow_right(int is_down);
-void scroll_arrow_up(int is_down);
-void scroll_arrow_down(int is_down);
+void scroll_arrow_left(int value);
+void scroll_arrow_right(int value);
+void scroll_arrow_up(int value);
+void scroll_arrow_down(int value);
```

Comment:
What should be passed as value here? Do we need constants/enum for those?

Reply:
> What should be passed as value here?

The joystick axis input, which starts from `0`, has a dead zone of `2000` and then goes up to `32768`.